app/views/accounts/show.html.erb

```diff
@@ -21,6 +21,10 @@
     <% if @presenter.show_service_provider_continue_partial? %>
       <%= render 'accounts/service_provider_continue', presenter: @presenter %>
     <% end %>
+
+    <% if @presenter.show_account_reset_requested_partial? %>
+      <%= render 'accounts/account_restored' %>
```

Comment:
This will persist as long as the account request exists. Are we expecting this to stay on their page forever, or will we clear out the account reset request after that first login/visit to the account page?

Reply:
ah, we should clear out the request after first visit.